File: POO/src/poo/Secretaria.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package poo;

/**
 *
 * @author Icaro
 */
public class Secretaria extends Funcionario {

    @Override
    public double calcularParticipacaoNosLucros() {
        return getSalario() * 0.2;
    }
}

File: POOC/POOC/br.com.funcionario/Secretaria.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Func
{
    class Secretaria : Funcionario
    {
        
        public override double calcularParticipacaoNosLucros() {
            return getSalario() * 0.2;
        }
    }
}
